lib/Doctrine/deploy/sampleData/ServiceEndpoints.xml

```diff
@@ -285,11 +285,385 @@
+        <SERVICE_TYPE>ARC-CE</SERVICE_TYPE>
+        <HOST_IP>188.500.50.99</HOST_IP>
+        <SCOPE>EGI</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>Y</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>PtujRPC</SITENAME>
+        <COUNTRY_NAME>Slovenia</COUNTRY_NAME>
+        <COUNTRY_CODE>SI</COUNTRY_CODE>
+        <ROC_NAME>NGI_SI</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="1142G0">
+        <PRIMARY_KEY>1142G0</PRIMARY_KEY>
+        <HOSTNAME>bdii.budapestinstitute-gridoperations.hu</HOSTNAME>
+        <GOCDB_PORTAL_URL>https://testing.host.com/portal/index.php?Page_Type=View_Object&amp;object_id=1142&amp;grid_id=0</GOCDB_PORTAL_URL>
+        <DESCRIPTION>Site BDII</DESCRIPTION>
+        <SERVICE_TYPE>Site-BDII</SERVICE_TYPE>
+        <HOST_IP>177.23.900.02</HOST_IP>
+        <SCOPE>EGI</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>N</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>Budapest Institute</SITENAME>
+        <COUNTRY_NAME>Hungary</COUNTRY_NAME>
+        <COUNTRY_CODE>HU</COUNTRY_CODE>
+        <ROC_NAME>NGI_HU</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="10098G0">
+        <PRIMARY_KEY>10098G0</PRIMARY_KEY>
+        <HOSTNAME>arc-ce.budapestinstitute-gridoperations.hu</HOSTNAME>
+        <GOCDB_PORTAL_URL>https://testing.host.com/portal/index.php?Page_Type=View_Object&amp;object_id=10098&amp;grid_id=0</GOCDB_PORTAL_URL>
+        <DESCRIPTION>arc-ce for site</DESCRIPTION>
+        <SERVICE_TYPE>ARC-CE</SERVICE_TYPE>
+        <HOST_IP>184.703.44.55</HOST_IP>
+        <SCOPE>EGI</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>Y</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>Budapest Institute</SITENAME>
+        <COUNTRY_NAME>Hungary</COUNTRY_NAME>
+        <COUNTRY_CODE>HU</COUNTRY_CODE>
+        <ROC_NAME>NGI_HU</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="2045G0">
+        <PRIMARY_KEY>2045G0</PRIMARY_KEY>
+        <HOSTNAME>local-lfc.budapestinstitute-gridoperations.hu</HOSTNAME>
+        <GOCDB_PORTAL_URL>https://testing.host.com/portal/index.php?Page_Type=View_Object&amp;object_id=2045&amp;grid_id=0</GOCDB_PORTAL_URL>
+        <DESCRIPTION>Local LFC</DESCRIPTION>
+        <SERVICE_TYPE>Local-LFC</SERVICE_TYPE>
+        <HOST_IP>189.400.55.12</HOST_IP>
+        <SCOPE>Local</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>Y</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>Budapest Institute</SITENAME>
+        <COUNTRY_NAME>Hungary</COUNTRY_NAME>
+        <COUNTRY_CODE>HU</COUNTRY_CODE>
+        <ROC_NAME>NGI_HU</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="3133G0">
+        <PRIMARY_KEY>3133G0</PRIMARY_KEY>
+        <HOSTNAME>SRM.budapestinstitute-gridoperations.hu</HOSTNAME>
```

Comment:
perhaps change 'institute' to some synonym - here and elsewhere.

Reply:
ok